installer/scripts/tue-install-impl

```diff
@@ -363,6 +381,35 @@ function tue-install-system
     TUE_INSTALL_SYSTEMS="$1 $TUE_INSTALL_SYSTEMS"
 }
 
+function tue-install-system-now
+{
+    tue-install-debug "tue-install-system-now $@"
+    if [ -z "$1" ]
+    then
+        tue-install-error "Invalid tue-install-system-now call: needs package as argument."
+    fi
+    local install_required=false
+
+    for pkg in "$@"
+    do
+        # Check if pkg is not already installed dpkg -S does not cover previously removed packages
+        # Based on https://stackoverflow.com/questions/1298066
+        if ! dpkg-query -W -f='${Status}' $pkg 2>/dev/null | grep -q "ok installed"
+        then
+            tue-install-debug "Package $pkg requires installation"
+            install_required=true
```

Comment:
It would be nicer to create a list of pkgs which are not installed yet. And then check if the list is empty. That is also the way it is done in tue-install-system.